joss/paper.md

```diff
@@ -0,0 +1,80 @@
+---
+title: 'MParT: Monotone Parameterization Toolkit'
```

Comment:
title is different from above in index.rst

Reply:
I'll change the index.rst title to match.